src/scores/continuous/standard_impl.py

```diff
@@ -418,3 +419,147 @@ def pbias(
 
     _pbias = 100 * error.mean(dim=reduce_dims) / obs.mean(dim=reduce_dims)
     return _pbias
+
+
+def kge(
+    fcst: xr.DataArray,
+    obs: xr.DataArray,
+    *,
+    reduce_dims: Optional[FlexibleDimensionTypes] = None,
+    preserve_dims: Optional[FlexibleDimensionTypes] = None,
+    scaling_factors: Optional[Union[list[float], np.ndarray]] = None,
+    return_components: bool = False,
+) -> XarrayLike:
+    # pylint: disable=too-many-locals
+    """
+    Calculate the Kling-Gupta Efficiency (KGE) between observed and simulated (or forecast) values.
+
+    KGE is a performance metric that decomposes the error into three components:
+    correlation, variability, and bias.
+    It is computed as:
+
+    .. math::
```

Comment:
<img width="583" alt="image" src="https://github.com/user-attachments/assets/a27cc2b7-b5d0-4708-bb00-6ebcac84575a">


This is how the mathjax is rendering now when I build the docs.